app/views/govuk_publishing_components/audit/show.html.erb

```diff
@@ -10,235 +10,21 @@
-              <h2 class="govuk-accordion__section-heading">
-                <span class="govuk-accordion__section-button" id="accordion-default-heading-2">
-                  Components containing components
-                </span>
-              </h2>
-              <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-2">
-                Shows which components contain other components
-              </div>
-            </div>
-            <div id="accordion-default-content-2" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
-              <dl class="govuk-summary-list">
-                <% @components[:components_containing_components].each do |component| %>
-                  <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">
-                      <a href="<%= component[:link] %>" class="govuk-link"><%= component[:component] %></a>
-                    </dt>
-                    <dd class="govuk-summary-list__value">
-                      <%= component[:sub_components].join(', ') %>
-                    </dd>
-                  </div>
-                <% end %>
-              </dl>
-            </div>
-          </div>
-          <div class="govuk-accordion__section ">
-            <div class="govuk-accordion__section-header">
-              <h2 class="govuk-accordion__section-heading">
-                <span class="govuk-accordion__section-button" id="accordion-default-heading-2">
-                  Components by application
-                </span>
-              </h2>
-              <div class="govuk-accordion__section-summary govuk-body" id="accordion-with-summary-sections-summary-2">
-                Shows which applications use each component
-              </div>
-            </div>
-            <div id="accordion-default-content-2" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
-              <% if @components[:components_by_application].any? %>
-                <dl class="govuk-summary-list">
-                  <% @components[:components_by_application].each do |component| %>
-                    <div class="govuk-summary-list__row">
-                      <dt class="govuk-summary-list__key">
-                        <%= component[:component] %> (<%= component[:count] %>)
-                      </dt>
-                      <dd class="govuk-summary-list__value">
-                        <%= component[:list] %>
-                      </dd>
-                    </div>
-                  <% end %>
-                </dl>
-              <% else %>
-                <p class="govuk-body">Sorry, no applications found.</p>
-              <% end %>
-            </div>
-          </div>
-        </div>
-      <% else %>
-        <p class="govuk-body">No components found.</p>
-      <% end %>
-    </div>
-  </div>
+<% if @in_application or @other_applications %>
```

Comment:
I've been reviewing this commit by commit, and I now see this commit fixes the problem with this page that I've already commented on. Could this commit be squashed into that one?